src/main/java/org/sagebionetworks/bridge/validators/StudyValidator.java

```diff
@@ -141,10 +146,15 @@ public void validate(Object object, Errors errors) {
             if (customEvent.getUpdateType() == null) {
                 errors.rejectValue("updateType", CANNOT_BE_NULL);
             }
+            scheduledCustomEventIds.remove(customEvent.getEventId());
             errors.popNestedPath();
         }
         if (uniqueIds.size() > 0 && (uniqueIds.size() != study.getCustomEvents().size())) {
-            errors.rejectValue(CUSTOM_EVENTS_FIELD, "cannot contain duplidate event IDs");
+            errors.rejectValue(CUSTOM_EVENTS_FIELD, "cannot contain duplicate event IDs");
+        }
+        if (!scheduledCustomEventIds.isEmpty()) {
+            errors.rejectValue(CUSTOM_EVENTS_FIELD, 
+                    String.format("cannot remove custom events currently used in a schedule: %s", scheduledCustomEventIds.toString()));
```

Comment:
You need to verify that the existing scheduleCustomEventIds are all in the study...but there could be none and that's okay (schedules can be saved with no sessions). Adding is okay here. Just not removing.

Reply:
Changed to set comparison operations.